### PR TITLE
Removes suffix from container networking plugins image name in mirroring

### PR DESCRIPTION
**Pull request by dougbtv**
Container networking CNI plugins images have recently been condensed into a single image, and the image name has changed. Previously I had made a change to condense them, however -- I missed the `-supported` suffix on the image name.

@smarterclayton -- Sorry for the additional fix here! Thanks.

### CODE REVIEW

**Comment by smarterclayton** [comment]
Did you already make the change to the OCP configs in ocp-build-data?

**Comment by dougbtv** [comment]
@smarterclayton -- we did, it was made in this change: https://gitlab.cee.redhat.com/openshift-art/ocp-build-data/merge_requests/172/diffs

**Comment by dougbtv** [comment]
Anything else needed here @smarterclayton ? Happy to do so if need be. We have a companion PR for this one for the cluster-network-operator https://github.com/openshift/cluster-network-operator/pull/237 that relies on this change. Thanks for sure.

**Comment by dougbtv** [comment]
@smarterclayton -- thanks for the pointer for the `./ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml` -- pushed an amended commit with changes there too.

File: ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml [hunk 1]
```diff
@@ -11,12 +11,9 @@ build_root:
...
 canonical_go_repository: github.com/openshift/containernetworking-plugins
 images:
-- dockerfile_path: images/supported/Dockerfile
+- dockerfile_path: Dockerfile
   from: base
-  to: container-networking-plugins-supported
-- dockerfile_path: images/unsupported/Dockerfile
-  from: base
-  to: container-networking-plugins-unsupported
+  to: container-networking-plugins
```
Comment (smarterclayton) [inline ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml]: You also need to change release-4.3 and release-4.2 versions

Comment (dougbtv) [reply ci-operator/config/openshift/containernetworking-plugins/openshift-containernetworking-plugins-master.yaml]: Thanks, got a new commit pushed up with changes to the `-release-4.2.yaml` and `-release-4.3.yaml` files

**Comment by dougbtv** [comment]
@smarterclayton -- I've got the changes in per your review (thanks for the eyes on that one). I'm having trouble getting the release-4.3 rehearsal test to pass, the master & release-4.2 rehearsal tests pass. Is the release-4.3 known to be flaky?

**Comment by dougbtv** [comment]
rebased to resolve conflicts.

**Comment by smarterclayton** [comment]
/lgtm

But be sure to watch the e2e-AWS builds and make sure this rename doesn’t affect them.

**Pull request merged**